KitePrintSDK/src/main/java/ly/kite/TemplateClass.java

```diff
@@ -0,0 +1,91 @@
+/*****************************************************
+ *
+ * TemplateClass.java
+ *
+ *
+ * Modified MIT License
+ * 
+ * Copyright (c) 2010-2015 Kite Tech Ltd. https://www.kite.ly
+ * 
+ * Permission is hereby granted, free of charge, to any person obtaining a copy
+ * of this software and associated documentation files (the "Software"), to deal
+ * in the Software without restriction, including without limitation the rights
+ * to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+ * copies of the Software, and to permit persons to whom the Software is
+ * furnished to do so, subject to the following conditions:
+ * 
+ * The software MAY ONLY be used with the Kite Tech Ltd platform and MAY NOT be modified
+ * to be used with any competitor platforms. This means the software MAY NOT be modified 
+ * to place orders with any competitors to Kite Tech Ltd, all orders MUST go through the
+ * Kite Tech Ltd platform servers. 
+ * 
+ * The above copyright notice and this permission notice shall be included in
+ * all copies or substantial portions of the Software.
+ * 
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NON INFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+ * THE SOFTWARE.
+ * 
+ *****************************************************/
+
+///// Package Declaration /////
+
+package ly.kite;
+
+
+///// Import(s) /////
+
+
+///// Class Declaration /////
+
+/*****************************************************
+ * 
+ * This class ...
+ * 
+ *****************************************************/
+public class TemplateClass

```

Comment:
Yes - it's the template for every new class I create. It doesn't 
necessarily have to be in the Git tree - I could just keep it on my 
filesystem - but if anyone else wants to create a new class it is useful 
to keep it in there.

On 20/07/15 10:29, Deon Botha wrote:
>
> In KitePrintSDK/src/main/java/ly/kite/TemplateClass.java 
> <https://github.com/OceanLabs/Android-Print-SDK/pull/9#discussion_r34979250>:
>
> > +///// Package Declaration /////
> > +
> > +package ly.kite;
> > +
> > +
> > +///// Import(s) /////
> > +
> > +
> > +///// Class Declaration /////
> > +
> > +/*****************************************************
> > + *
> > + * This class ...
> > + *
> > + *****************************************************/
> > +public class TemplateClass
>
> Do you use this class? Might be worth deleting it entirely if not?
>
> —
> Reply to this email directly or view it on GitHub 
> <https://github.com/OceanLabs/Android-Print-SDK/pull/9/files#r34979250>.
>